lib/teams.rb

```diff
@@ -0,0 +1,19 @@
+class Teams 
+  attr_reader :team_id, 
+              :teamName
+
+
+  def initialize(team_id, teamName)
+    @team_id = team_id
+    @teamName = teamName
+  end
+
+  def self.load_csv(file_path)
+    teams = []
```

Comment:
Instead of a teams array being returned here, I would instance create a class variable that stores all these instances. That way we can have another method that returns said class variable. Otherwise we could only ever access this data by creating it over and over again. Which could increase our runtimes.

Key points:
- class variable (likely named @@instances or @@teams)
- a method (.all) that returns this class variable